xml2rfc/data/xml2rfc.css

```diff
@@ -16,6 +16,12 @@
 @import url('https://fonts.googleapis.com/css?family=Noto+Serif'); /* Serif (print) */
 @import url('https://fonts.googleapis.com/css?family=Roboto+Mono'); /* Monospace */
 
+:root {
+  --font-sans: 'Noto Sans', Arial, Helvetica, sans-serif;
+  --font-serif: 'Noto Serif', 'Times', 'Times New Roman', serif;
```

Comment:
Is it intentional to introduce two new fallback fonts (Times & Times New Roman) here?

Reply:
I'm doing that to be consistent with the fallbacks you have for the sans-serif variant.  This is a common fallback pattern.